internal/restapi/browse/handler.go

```diff
@@ -143,3 +150,98 @@ func (h *browseHttpService) getModuleMetadataHandler() http.Handler {
 		h.responseHandler.Write(rw, data, http.StatusOK)
 	})
 }
+
+// GetReleasesHandler will return a list of all releases published.
+func (h *browseHttpService) getReleasesHandler() http.Handler {
+	return http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
+		var maxAge uint64
+
+		parsedAge, err := time.ParseDuration(mux.Vars(r)["age"])
+		if err != nil || parsedAge.Seconds() < 3600 {
+			maxAge = 3600
+		} else {
+			maxAge = uint64(parsedAge.Seconds())
+		}
+
+		conn, err := services.CreateGRPCConnection(release.ReleaseServiceEndpoint)
+		if err != nil {
+			log.Printf("Failed to connect to '%s': %v", release.ReleaseServiceEndpoint, err)
```

Comment:
Instead of log lines lets use Otel traces as in the above getModuleMetadataHandler()

Reply:
In getModuleMetadataHandler() we are currently not using otel traces for errors. Did you mean we add `span.RecordError(err)` instead of the Log statement?

Reply:
Yes please